README.md

```diff
@@ -0,0 +1,117 @@
+# specfile
+
+Python library for parsing and manipulating RPM spec files. Main focus is on modifying existing spec files, any change should result in a minimal diff.
+
+## Caveats
+
+### RPM macros
+
+specfile uses RPM for parsing spec files and macro expansion. Unfortunately, macros are always stored in a global context, which poses a problem for multiple instances of Specfile.
+
+## Examples and use cases
```

Comment:
We should maybe mention for now that most of these have not been implemented yet and are just examples of the intended future interface

Reply:
Also maybe a sentence like `The following examples should cover use cases described by [packit](https://github.com/packit/research/blob/main/specfile/README.md)`